meme_generator.py

```diff
@@ -154,7 +158,7 @@ def random_meme(show='True'):
             preprocessImages(img1)
             preprocessImages(img2)
             formatObj = Format2(img1, img2, top_text, bottom_text)
-            formatObj.generate()
+            use(formatobj)
```

Comment:
There is a typo with some of the function calls with ``` formatobj ``` , Please make changes it will cause undefined variable  exception.
We can not test an interactive mode generation procedure with tests so please try not to make these typos. Please run interactive mode tests manually.